website/www/site/content/en/get-started/wordcount-example.md

```diff
@@ -1405,10 +1405,10 @@ messages from a Pub/Sub subscription or topic using
 {{< highlight py >}}
   # Read from Pub/Sub into a PCollection.
   if known_args.input_subscription:
-    lines = p | beam.io.ReadStringsFromPubSub(
+    lines = p | beam.io.ReadFromPubSub(
```

Comment:
An important difference is that ReadFromPubSub returns a `bytes` object (the raw message data).
I would:
- rename `lines` to `data`
- add an additional step below: `lines = data | 'DecodeString' >> beam.Map(lambda d: d.decode('utf-8')))`